Appsilon/shiny.gosling
# Ability to call the API methods

!! Before starting the review on this PR, let's close #13 Which is the base for this branch as well.

Closes #15 
Closes #9 

## Changes description

* Implementing a component (`customGosling`) wrapping the GoslingComponent from gosling.js.
* Implementing an API (`GoslingComponents`) as interface to Shiny.
* Added a method and an R function (`zoom_to_extent`) to call the corresponding method.
* Updated the Basic example in the dashboard adding a reset zoom button.

## How to test
* Execute the dashboard example `examples/dasboard/app.R` go to the Basic tab, zoom out or zoom in in any of the two examples and then click the reset zoom button. The zoom should be reset.
